spec/chromium-spec.js

```diff
@@ -223,6 +223,26 @@ describe('chromium feature', () => {
       b = window.open(`file://${fixtures}/pages/window-open-size.html`, '', 'show=no')
     })
 
+    for (const show in [true, false]) {
+      it(`inherits parent visibility over parent {show=${show}} option`, (done) => {
+        let w = new BrowserWindow({show: show})
+
+        // toggle visibility
+        if (show) {
```

Comment:
(nit) for something this short, maybe use a ternary?

Reply:
Tried it that way and wound up in an ad-hoc bikeshed session with Sam and Alexey with more opinions than people. Let's leave it as-is.

Reply:
sounds good to me 😁 already approved, ready whenever!